deb/openmediavault/usr/share/openmediavault/engined/inc/60filesystembackend.inc

```diff
@@ -24,7 +24,7 @@ namespace OMV\System\Filesystem\Backend;
 ///////////////////////////////////////////////////////////////////////////////
 // Register default filesystem backends.
 ///////////////////////////////////////////////////////////////////////////////
-$backends = [ "Btrfs", "Ext", "Ext2", "Ext3", "Ext4", "Hfsplus",
+$backends = [ "Bcache", "Btrfs", "Ext", "Ext2", "Ext3", "Ext4", "Hfsplus",
```

Comment:
You need to add your own file that is performing this action. This should not be added to the core code. Check https://github.com/OpenMediaVault-Plugin-Developers/openmediavault-mergerfs/blob/main/usr/share/openmediavault/engined/inc/59mergerfs.inc for example.

Reply:
OK, ignore this please. Makes sense to have this in the core package.